invokeai/app/services/config/config_default.py

```diff
@@ -512,3 +437,73 @@ def get_config() -> InvokeAIAppConfig:
         default_config.write_file(config.config_file_path, as_example=False)
 
     return config
+
+
+####################################################
+# VERSION MIGRATIONS
+####################################################
+
+
+@ConfigMigrator.register(from_version="0.0.0", to_version="4.0.0")
```

Comment:
In the migrator logic, we set the version to "3.0.0" if the schema version is missing and we are on a v3 config file, but here we register the migration as if it was from "0.0.0". This seems off

Reply:
Changed to 3.0.0 and tested.